modules/OCPPConfiguration/main/event_handler.hpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include "config_writer.hpp"
+#include "generated/types/ocpp.hpp"
+#include "mapping_reader.hpp"
+
+namespace module {
+
+class EventHandler {
+public:
+    EventHandler(const std::string& config_mapping_path, const std::string& user_config_path);
+    void handleEvent(const types::ocpp::EventData& event_data);
+
+private:
+    using EventMap = MappingReader::ConfigMap;
+    using EverestConfigPair = MappingReader::EverestConfigPair;
+
+    const EventHandler::EverestConfigPair find_event_in_map(const std::string& event_name) const;
```

Comment:
what if the event can't be found?  What will the pair look like?

Reply:
I think it should either throw an exception or just ignore it. It depends on the configuration of the module whether a variable should be in that map or not